src/test/java/io/confluent/connect/elasticsearch/integration/ElasticSearchSinkConnectorIT.java

```diff
@@ -0,0 +1,308 @@
+
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.json.JsonConverter;
+import org.apache.kafka.connect.json.JsonConverterConfig;
+import org.apache.kafka.connect.runtime.SinkConnectorConfig;
+import org.apache.kafka.connect.storage.ConverterConfig;
+import org.apache.kafka.connect.storage.ConverterType;
+import org.elasticsearch.common.Strings;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.apache.kafka.connect.runtime.ConnectorConfig.CONNECTOR_CLASS_CONFIG;
+import static org.apache.kafka.connect.runtime.ConnectorConfig.TASKS_MAX_CONFIG;
+
+@Category(IntegrationTest.class)
+public class ElasticSearchSinkConnectorIT extends BaseConnectorIT {
+
+  private static final Logger log = LoggerFactory.getLogger(ElasticSearchSinkConnectorIT.class);
+
+  private static final int NUM_RECORDS = 10000;
+  private Map<String, String> props;
+  private JsonConverter jsonConverter = new JsonConverter();
+
+  @Before
+  public void setup() {
+    startConnect();
+    connect.kafka().createTopic(KAFKA_TOPIC, 1);
+    Map<String, String> converterConfig = new HashMap<>();
+    converterConfig.put(JsonConverterConfig.SCHEMAS_CACHE_SIZE_CONFIG, "100");
+    converterConfig.put(ConverterConfig.TYPE_CONFIG, ConverterType.VALUE.getName());
+    converterConfig.put(JsonConverterConfig.SCHEMAS_ENABLE_CONFIG, "true");
+    jsonConverter.configure(converterConfig);
+    startEScontainer();
+    props = getSinkConnectorProperties();
+    super.setUp(props);
+  }
+
+  @After
+  public void close() {
+    stopConnect();
+    stopESContainer();
+  }
+
+  @Test
+  public void testSuccess() throws InterruptedException, IOException {
+    sendTestDataToKafka(0, NUM_RECORDS);
+    ConsumerRecords<byte[], byte[]> totalRecords = connect.kafka().consume(
+        NUM_RECORDS,
+        CONSUME_MAX_DURATION_MS,
+        KAFKA_TOPIC);
+    log.info("Number of records added in kafka {}", totalRecords.count());
+
+    // Configure Connector and wait some specific time to start the connector.
+    connect.configureConnector(CONNECTOR_NAME, props);
+    waitForConnectorToStart(CONNECTOR_NAME, Integer.valueOf(MAX_TASKS));
+
+    // Wait Connector to write data into elastic-search
```

Comment:
```suggestion
    // Wait Connector to write data into elasticsearch
```

This is single word and applicable everywhere.